content/research/papers.md

```diff
@@ -4,4 +4,147 @@ date: 2021-05-28T09:27:27-05:00
+
+
+**Boyle, Aoife, Sim, Stuart A., Hachinger, Stephan, et al.** 2017, A&A, *"Helium in double-detonation models of type Ia supernovae"* <a href="https://ui.adsabs.harvard.edu/abs/2017A&A...599A..46B" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Noebauer, U. M., Taubenberger, S., Blinnikov, S., et al.** 2016, MNRAS, *"Type Ia supernovae within dense carbon- and oxygen-rich envelopes: a model for `Super-Chandrasekhar' explosions?"* <a href="https://ui.adsabs.harvard.edu/abs/2016MNRAS.463.2972N" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Inserra, C., Bulla, M., Sim, S. A., et al.** 2016, ApJ, *"Spectropolarimetry of Superluminous Supernovae: Insight into Their Geometry"* <a href="https://ui.adsabs.harvard.edu/abs/2016ApJ...831...79I" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Szalai, Tamás, Vinkó, József, Nagy, Andrea P., et al.** 2016, MNRAS, *"The continuing story of SN IIb 2013df: new optical and IR observations and analysis"* <a href="https://ui.adsabs.harvard.edu/abs/2016MNRAS.460.1500S" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Magee, M. R., Kotak, R., Sim, S. A., et al.** 2016, A&A, *"The type Iax supernova, SN 2015H. A white dwarf deflagration candidate"* <a href="https://ui.adsabs.harvard.edu/abs/2016A&A...589A..89M" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Dubernet, M. L., Antony, B. K., Ba, Y. A., et al.** 2016, JPhB, *"The virtual atomic and molecular data centre (VAMDC) consortium"* <a href="https://ui.adsabs.harvard.edu/abs/2016JPhB...49g4003D" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Parrent, J. T., Howell, D. A., Fesen, R. A., et al.** 2016, MNRAS, *"Comparative analysis of SN 2012dn optical spectra: days -14 to +114"* <a href="https://ui.adsabs.harvard.edu/abs/2016MNRAS.457.3702P" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Young, P. R., Dere, K. P., Landi, E., et al.** 2016, JPhB, *"The CHIANTI atomic database"* <a href="https://ui.adsabs.harvard.edu/abs/2016JPhB...49g4009Y" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Noebauer, U. M., Sim, S. A.** 2015, MNRAS, *"Self-consistent modelling of line-driven hot-star winds with Monte Carlo radiation hydrodynamics"* <a href="https://ui.adsabs.harvard.edu/abs/2015MNRAS.453.3120N" target="_blank" rel="noopener nofollow">(ADS Link) </a>
+
+
+**Matthews, J. H., Knigge, C., Long, K. S., et al.** 2015, MNRAS, *"The impact of accretion disc winds on the optical spectra of cataclysmic variables"* <a href="https://ui.adsabs.harvard.edu/abs/2015MNRAS.450.3331M" target="_blank" rel="noopener nofollow">(ADS Link)</a>
+
+
+**Kerzendorf, Wolfgang E., Sim, Stuart A.** 2014, MNRAS, *"A spectral synthesis code for rapid modelling of supernovae"* <a href="https://ui.adsabs.harvard.edu/abs/2014MNRAS.440..387K" target="_blank" rel="noopener nofollow">(ADS Link)</a>
```

Comment:
Add section for carsus package and fill in with item no. 1 from here: https://tardis-sn.github.io/carsus/research/publications.html (no. 2 is actually a poster so we don't want it here)